python_modules/dagster/dagster/_core/definitions/events.py

```diff
@@ -421,7 +422,23 @@ def label(self) -> str:
         return " ".join(self.asset_key.path)
 
 
-@whitelist_for_serdes(old_storage_names={"Materialization"})
+UNDEFINED_ASSET_KEY = AssetKey(["undefined"])
```

Comment:
should this be more obnoxious ie `_undefined_` / `__undefined__` / `<<undefined>>` ? 
 
unclear how many of these flowed through the system so not sure this is worth fretting about at all

Reply:
I'm partial to  `__undefined__`, why not